tests/Microsoft.Identity.Test.Integration.netfx/SeleniumTests/InteractiveFlowTests.cs

```diff
@@ -227,6 +239,16 @@ private async Task<AuthenticationResult> RunTestForUserAsync(LabResponse labResp
                     .WithTestLogging()
                     .Build();
             }
+            else if (multiCloudSupport)
+            {
+                pca = PublicClientApplicationBuilder
+                    .Create(labResponse.App.AppId)
+                    .WithRedirectUri(SeleniumWebUI.FindFreeLocalhostRedirectUri())
+                    .WithMultiCloudSupport()
```

Comment:
You can add just the "WithMultiCloudSupport(multiCloudSupport)" to the existing builder.